Tutorial.md

```diff
@@ -0,0 +1,121 @@
+# Tutorial on Show Design
+
+## Overview
+
+Sparkle Motion is a tool to design and run a light show.
+
+First, you define a "scene" -- an arrangement of physical light fixtures and their controller mappings.
+
+Next, you design a "show" -- a set of visual effects to project on a scene.
+
+Visual effects programmed as "shaders" -- small GLSL programs that process inputs like xy coordinates from the scene, colors from another shader, and audio metadata from the sound analysis plugin, and generate outputs like another xy coordinate for distortion or projection mapping, or a color for a light.
+
+Finally, you use the designer mode to design a UI for all the "controls" and run the light show in real time.
+
+## Simulator Quick Start
+
+On OS X, start the simulator:
+
+```
+brew install openjdk
+export PATH="/opt/homebrew/opt/openjdk/bin:$PATH"
+./gradlew --continuous jsRun
+```
+
+A brand new session starts with no Show or Scene loaded, and the Show/Scene menu open. For an existing session click the "☰" icon to open the Show/Scene menu.
+
+## Scene Quick Start
+
+Load a scene from the BAAAHS template. Defining a scene's model, light controllers and fixtures is an exercise for another day...
+
+### Load the BAAAHS template
+
+- "☰" to open the Show/Scene menu then Scene tab
+- "+ New Scene"
+- From template "BAAAHS" to see the sheep model 🐑
+- "Save" button, enter "my-baaahs", then "save"
+
+## Show Quick Start
+
+Next we'll create an empty show, define a default projection, then program our first effect.
+
+### Create an empty show
+
+- "☰" to open the Show/Scene menu then Show tab
+- "+ New Show"
+- "Empty Show"
+- "Save" button, enter "my-baaahs", then "save"
+
+### Define a default projection
+
+- "☰" to open the Show/Scene menu then Show tab
+- Toggle "Design Mode"
+- "✎" next to the show name
+- New Projection Shader
+- Shader name: "Flat Projection"
+- Use the default code
+
+  ```c
```

Comment:
`glsl` probably works here too.

Reply:
`glsl` this doesn't work in vscode out of the box but it does work on GitHub https://gist.github.com/nzoschke/937e91c0fd2fefb7e1fccb8ba5813a56

Do you have a preference either way?

Reply:
Not particularly, it'll do glsl keyword highlighting but no strong preference.

Reply:
going to stich with `c` for now